CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ This is the changelog for SpotBugs. This follows [Keep a Changelog v1.0.0](http:
 Currently the versioning policy of this project follows [Semantic Versioning v2.0.0](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased - 2021-??-??
+* Rule `MOE_METHOD_INVOKED_IN_OPTIONAL_ORELSE` covers method invocation in [Optional.orElse](https://www.baeldung.com/java-optional-or-else-vs-or-else-get) ([#1808](https://github.com/spotbugs/spotbugs/pull/1808))
```

Comment:
This is a new feature, so describe the change in `### Added` chapter. Refer [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) for detail.

```suggestion
### Added
* New rule `MOE_METHOD_INVOKED_IN_OPTIONAL_ORELSE` which detects method invocation in [Optional#orElse()](https://www.baeldung.com/java-optional-or-else-vs-or-else-get) ([#1808](https://github.com/spotbugs/spotbugs/pull/1808))
```

Reply:
we have conflict in this file. Please merge the latest `master` branch to your working branch, and resolve conflict to make it mergeable.

Reply:
done